apps/backend/src/lib/controller/chip/managed/client.ts

```diff
@@ -18,9 +19,11 @@ import("@/lib/controller/chip/managed/tap");
 
 export class ManagedChipClient implements iChipClient {
   prismaClient: PrismaClient;
+  notificationClient: TelegramNotificationClient;
```

Comment:
This seems redundant as the client is already included in the Controller. You can remove this if you move the notification here: https://github.com/cursive-team/connections/blob/main/apps/backend/src/lib/controller/index.ts#L141